From kristen.c.accardi@intel.com Wed Nov  2 19:43:44 2005
Subject: [patch 3/4] pci: use pin stored in pci_dev
From: Kristen Accardi <kristen.c.accardi@intel.com>
To: <rajesh.shah@intel.com>, <greg@kroah.com>, <len.brown@intel.com>
Date: Wed, 02 Nov 2005 16:24:39 -0800
Message-Id: <1130977479.8321.41.camel@whizzy>

Use the stored value of the interrupt pin rather than try to read
the config again.

Signed-off-by: Kristen Carlson Accardi <kristen.c.accardi@intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
 
---
 drivers/pci/pci.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/pci/pci.c
+++ gregkh-2.6/drivers/pci/pci.c
@@ -587,7 +587,7 @@ pci_get_interrupt_pin(struct pci_dev *de
 {
 	u8 pin;
 
-	pci_read_config_byte(dev, PCI_INTERRUPT_PIN, &pin);
+	pin = dev->pin;
 	if (!pin)
 		return -1;
 	pin--;
